README.md

```diff
@@ -17,37 +17,61 @@ npm install @wert-io/module-react-component
 ## Usage example
 
 ```
-import WertModule from '@wert-io/module-react-component';
+import { useWertWidget } from '@wert-io/module-react-component';
 
-class App extends React.Component {
-  constructor(props) {
-    super(props);
-
-    this.state = {
-      wertOptions: {
-        partner_id: '{{YOUR_PARTNER_ID}}',
+export default function WidgetButton () {
+    const options = {
+        partner_id: 'default',
         listeners: {
-          loaded: () => console.log('loaded'),
+            loaded: () => console.log('loaded'),
         },
+    };
 
-        ...
+    const { mountWertWidget } = useWertWidget(options);
 
-      },
-    };
-  }
-
-  render() {
-    return (
-      <div className="app">
-        <WertModule
-          options={this.state.wertOptions}
-        />
-      </div>
-    );
-  }
+    return <button onClick={() => mountWertWidget()}>Make A Purchase</button>
 }
 ```
 
-You can find the Whole list of options [here](https://www.npmjs.com/package/@wert-io/widget-initializer#documentation).
+You can find the full list of the options that can be passed to the widget [here](https://www.npmjs.com/package/@wert-io/widget-initializer#documentation).
+
+### Events
+
+To get the whole list of available events, you can use a static property `eventTypes`:
+
+```
+  const { eventTypes } = useWertWidget(wertOptions);
+  console.log(eventTypes);
+```
+
+You can read the events descriptions [here](https://www.npmjs.com/package/@wert-io/widget-initializer#listeners).
+
+
+### Methods
+
+| Method                 | Description                   |
+|------------------------|-------------------------------|
+| **mountWertWidget**    | Mounts module in DOM          |
```

Comment:
What do you think about the naming of the methods? I thought that keeping them as they are in the widget initializer (mount, setTheme) would be inconvenient for the end user (they may have other methods with similar naming), so they'll have to rename it anyway